### PR TITLE
(1) Add possibility to undo ftplugin and (2) only run NERDcommenter specific options when it is loaded

### PR DESCRIPTION
**Pull request by a-vrma**
1. This lets people change filetypes within the window
    with `setfiletype`. Also update the last changed date.
2. This is a convention used in the vim runtime files so that people who
    don't use those specific plugins don't have their namespace polluted.

### CODE REVIEW

**Comment by a-vrma** [comment]
The nerd commenter specific options might have to be removed if we want to upstream this, but I'm not sure.

**Comment by a-vrma** [comment]
> what do you think about just deleting all the NERDCommenter stuff instead?

I think that's a great idea. I force-pushed a change that removes NERDCommenter entirely.
I noticed that the ftplugin doesn't have the `comments` option set so I added that.

File: ftplugin/toml.vim [hunk 1]
```diff
@@ -10,24 +10,10 @@ let b:did_ftplugin = 1
...
-    endif
-elseif exists('g:NERDCustomDelimiters')
-    if !has_key(g:NERDCustomDelimiters, 'toml')
-        let g:NERDCustomDelimiters.toml = s:delims
-    endif
-else
-    let g:NERDCustomDelimiters = { 'toml': s:delims }
-endif
-unlet s:delims
+setlocal comments=:#
```
Comment (cespare) [inline ftplugin/toml.vim]: Why is the `:` here?

Comment (a-vrma) [reply ftplugin/toml.vim]: That's how the `comments` option works. From the vim help:

> The 'comments' option is a comma-separated list of parts.  Each part defines a
> type of comment string.  A part consists of:
>     {flags}:{string}
> ...
> When a string has none of the 'f', 's', 'm' or 'e' flags, Vim assumes the
> comment string repeats at the start of each line.  The flags field may be
> empty.

Comment (cespare) [reply ftplugin/toml.vim]: Thanks! Looks like `format-comments` was the key section.

**Pull request merged**